### PR TITLE
Switch to page meta object

### PR DESCRIPTION
**Pull request by anthmatic**
Switches the base template to use the `page_met`a object for meta fields since those contain information about overrides coming from the CMS.

### CODE REVIEW

File: src/templates/layouts/base.html [hunk 1]
```diff
@@ -2,9 +2,10 @@
 <html lang="{{ html_lang }}" {{ html_lang_dir }}>
   <head>
     <meta charset="utf-8">
-    <title>{{ content.html_title }}</title>
+    <title>{{ page_meta.html_title }}</title>
     {% if site_settings.favicon_src %}<link rel="shortcut icon" href="{{ site_settings.favicon_src }}" />{% endif %}
-    <meta name="description" content="{{ content.meta_description }}">
+    <meta name="description" content="{{ page_meta.meta_description }}">
+    <link rel="canonical" href="{{ page_meta.canonical_url }}">
```
Comment (williamspiro) [inline src/templates/layouts/base.html]: I think including this means two canonical tags will be printed, which represents a confusing piece of magic. cc @nbelisle11

**Comment by nbelisle11** [comment]
@williamspiro I believe that's correct. Canonical urls get added in the renderer. Also, there are a lot of cases where you wouldn't want a canonical url on your page at all.

File: src/templates/layouts/base.html (hunk 1 above)
Comment (nbelisle11) [reply src/templates/layouts/base.html]: @williamspiro I believe that's correct. Canonical urls get added in the renderer. Also, there are a lot of cases where you wouldn't want a canonical url on your page at all.

Comment (anthmatic) [reply src/templates/layouts/base.html]: Ah, good catch. I tested this on a page that doesn't set a canonical tag. Will remove it.

**Pull request merged**